### PR TITLE
Bump version of eks-operator dependency

### PR DESCRIPTION
**Pull request by thedadams**
There was a bug in eks-operator and that function is also used here. We bump the version of eks-operator to include the bug fix.

Issue:
https://github.com/rancher/rancher/issues/29949

### CODE REVIEW

**Review by rmweir — COMMENTED** [review]
Did you run `go mod tidy`?

**Comment by thedadams** [comment]
I don't remember if I did it explicitly for this branch, but running it now doesn't change anything.

File: go.mod [hunk 1]
```diff
@@ -82,7 +82,7 @@ require (
 	github.com/prometheus/common v0.10.0
 	github.com/rancher/apiserver v0.0.0-20201023000256-1a0a904f9197
 	github.com/rancher/dynamiclistener v0.2.1-0.20200910203214-85f32491cb09
-	github.com/rancher/eks-operator v0.1.0-rc30
+	github.com/rancher/eks-operator v1.0.4
```
Comment (cbron) [inline go.mod:85]: Needs another change in pkg/apis/go.mod and then run go mod tidy in both.

**Review by cbron — CHANGES_REQUESTED** [review]

**Review by rmweir — APPROVED** [review]
LGTM

**Review by cbron — DISMISSED** [review]

**Review by cbron — APPROVED** [review]

**Pull request merged**